todo.js

```diff
@@ -0,0 +1,26 @@
+const addTaskBtn = document.getElementById("addTask");
+const taskInputText = document.getElementById("taskInput");
+const taskList = document.getElementById("taskList");
+
+let todos = [];
+
+const resetTaskInput = () => {
+  taskInputText.value = "";
+};
+const createTask = () => {
```

Comment:
`renderTodoList` is a more appropriate name as the function renders the list for us.
```suggestion
const renderTodoList = () => {
```